src/main/scala/com/gu/contentapi/firehose/kinesis/KinesisStreamReader.scala

```diff
@@ -0,0 +1,53 @@
+package com.gu.contentapi.firehose.kinesis
+
+import java.util.concurrent.Executors
+
+import com.amazonaws.auth.AWSCredentialsProvider
+import com.amazonaws.services.kinesis.metrics.impl.NullMetricsFactory
+import com.google.common.util.concurrent.ThreadFactoryBuilder
+import java.util.UUID
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.{ Worker, KinesisClientLibConfiguration, InitialPositionInStream }
+import com.amazonaws.services.kinesis.clientlibrary.interfaces.IRecordProcessorFactory
+
+trait KinesisStreamReader {
+
+  val streamName: String
```

Comment:
Yeah I started doing this actually and then thought I'd wait to see what people thought at review. Will make that change.